doc/examples/filters/plot_entropy.py

```diff
@@ -12,14 +12,19 @@
 
 In the first example, the image is composed of two surfaces with two slightly
 different distributions. The image has a uniform random distribution in the
-range [-14, +14] in the middle of the image and a uniform random distribution in
-the range [-15, 15] at the image borders, both centered at a gray value of 128.
+range [-15, +15] in the middle of the image and a uniform random distribution in
+the range [-14, 14] at the image borders, both centered at a gray value of 128.
 To detect the central square, we compute the local entropy measure using a
 circular structuring element of a radius big enough to capture the local gray
 level distribution. The second example shows how to detect texture in the camera
 image using a smaller structuring element.
 
 """
+
+##################
```

Comment:
I think the length of the line of `#'` needs to be longer for sphinx-gallery to detect that it's a new block of text. See the artifact https://162-2014929-gh.circle-artifacts.com/0/doc/build/html/auto_examples/filters/plot_entropy.html#sphx-glr-auto-examples-filters-plot-entropy-py where the two figures are together at the top.